repository/h.json

```diff
@@ -828,8 +828,8 @@
 		},
 		{
 			"name": "HoogleSearch",
-			"details": "https://github.com/s4wny/HoogleSearch",
-			"author": "Sawny",
+			"details": "https://github.com/pennychase/HoogleSearch",
+			"author": "Penny Chase",
```

Comment:
If you want to keep the existing repo url in "details" you may still want to update "author" here.